generators/environment/templates/bin/start.sh

```diff
@@ -54,6 +55,7 @@ while true ; do
     dev|int|local|ms|qa|review) DOCKER_ENV=$1 ; shift ;;
     -e|--environment) DOCKER_ENV=$2 ; shift 2 ;;
     -i|--no-validate) NO_VALIDATE=" --no-validate"; shift ;;
+    -f|--force) FORCE=" --force"; shift ;;
```

Comment:
Hey this works! I was thinking more in the documentation realm. Since this --force is specific to only the install operation, we should probably echo out a statement such as "The start.sh --force flag was used, site install will proceed despite any errors."